front/src/applications/stdcm/hooks/useConflicts.ts

```diff
@@ -0,0 +1,108 @@
+import { useEffect, useState } from 'react';
+
+import type { TFunction } from 'i18next';
+
+import type { ManageTrainSchedulePathProperties } from 'applications/operationalStudies/types';
+import { processConflicts } from 'applications/stdcm/utils/fomatConflicts';
+import type { Conflict } from 'common/api/osrdEditoastApi';
+
+const useConflictsMessages = (
+  t: TFunction,
+  conflictData?: Conflict[],
+  pathProperties?: ManageTrainSchedulePathProperties
+) => {
+  const [trackConflicts, setTrackConflicts] = useState<string[]>([]);
+  const [workConflicts, setWorkConflicts] = useState<string[]>([]);
+
+  useEffect(() => {
+    if (!pathProperties || !conflictData) return;
```

Comment:
After the function parameter comment

```suggestion
    if (!selectedSimulationOutput.conflicts) return;
```